src/telegram.js

```diff
@@ -845,6 +845,157 @@ class TelegramBot extends EventEmitter {
   }
 
   /**
+   * Use this method to restrict a user in a supergroup.
+   * The bot must be an administrator in the supergroup for this to work
```

Comment:
Avoid verbosity in doc comments since we have already linked the method to the telegram documentation.